src/poliastro/twobody/orbit.py

```diff
@@ -610,6 +610,71 @@ def parabolic(
         ss = ClassicalState(attractor, p, 1.0 * u.one, inc, raan, argp, nu)
         return cls(ss, epoch, plane)
 
+    @classmethod
+    @u.quantity_input(alt=u.m, inc=u.rad, argp=u.rad, raan=u.rad, arglat=u.rad)
+    def frozen(
+        cls,
+        alt,
+        inc=Earth.critical_inclinations[0],
+        argp=Earth.critical_argps[0],
+        raan=0 * u.deg,
+        arglat=0 * u.deg,
+        epoch=J2000,
+        plane=Planes.EARTH_EQUATOR,
+    ):
+        """Return frozen 'Orbit'.
+
+        Parameters
+        ----------
+        alt : ~astropy.units.Quantity
+            Altitude over surface.
+        inc : ~astropy.units.Quantity, optional
+            Inclination, default to critical value.
+        argp : ~astropy.units.Quantity, optional
+            Argument of the pericenter, default to critical value.
+        raan : ~astropy.units.Quantity, optional
+            Right ascension of the ascending node, default to 0 deg.
+        arglat : ~astropy.units.Quantity, optional
+            Argument of latitude, default to 0 deg.
+        epoch: ~astropy.time.Time, optional
+            Epoch, default to J2000.
+        plane : ~poliastro.frames.Planes
+            Fundamental plane of the frame.
+        """
+        if alt <= 0:
+            raise ValueError(
+                "The semimajor axis may not be smaller that the earth's radius"
+            )
+        a = Earth.R + alt
```

Comment:
Frozen orbits can also appear in other planets with a non uniform gravitational field, therefore we should not make this method Earth specific. If the `attractor` doesn't have J2 and J3, we can just show the corresponding `AttributeError` to the user.